frontend/src/views/CellGuide/components/CellGuideCard/components/MarkerGeneTables/index.tsx

```diff
@@ -357,12 +360,18 @@ const MarkerGeneTables = ({
           onClick={() => setGeneInfoGene(row.symbol.toUpperCase())}
         />
         {showEye && (
-          <StyledIconImage
-            data-testid={MARKER_GENES_TREE_ICON_BUTTON_TEST_ID(row.symbol)}
+          <StyledImageWrapper
```

Comment:
a parent element for the icon image with min-width and min-height set was needed to prevent the image from resizing smaller when the gene symbol is too long